galasa-ui/src/app/actions/deleteUserFromService.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+"use server";
+
+import { UsersAPIApi } from "@/generated/galasaapi";
+import * as Constants from "@/utils/constants";
+import { createAuthenticatedApiConfiguration } from '@/utils/api';
+
+
+export const deleteUserFromService = async (userNumber: string) => {
+
+  if (!userNumber) {
+    throw new Error('User Number is required');
+  }
+
+  const apiConfig = createAuthenticatedApiConfiguration();
+  const usersApiClient = new UsersAPIApi(apiConfig);
+
+  try {
+
+    await usersApiClient.deleteUserByNumber(userNumber, Constants.CLIENT_API_VERSION);
+    return { status: 204, message: "User deleted." };
```

Comment:
Do we need to return HTTP status codes in server actions? If not, we can remove them and not have this return anything (acting like a void function which could throw an error)